en_US/data-integration/data-bridge-kafka.md

```diff
@@ -103,13 +103,13 @@ Before adding a Kafka Sink action, you need to create a Kafka producer connector
 
      See [Authentication Method](#authentication-method) for details on each method.
 
-   - Leave other options as default or configure them according to your business needs.
+   - If you want to establish an encrypted connection, click the **Enable TLS** toggle switch. For more information about TLS connections, see [TLS for External Resource Access](../network/overview.md#tls-for-external-resource-access).
 
-   - If you want to establish an encrypted connection, click the **Enable TLS** toggle switch. For more information about TLS connection, see [TLS for External Resource Access](../network/overview.md#tls-for-external-resource-access).
+   - **Advanced Settings** (optional): See [Advanced Configurations](#advanced-configurations).
```

Comment:
We have "Advanced Configuration" for "Azure Event Hub", and "Advanced Configurations" (plural) here. May be stick to "Advanced Configuration" (singular) everywhere?